Kernel/GlobalProcessExposed.cpp

```diff
@@ -55,10 +55,16 @@ class ProcFSAdapters final : public ProcFSGlobalInformation {
                 auto ipv4_netmask = TRY(adapter.ipv4_netmask().to_string());
                 TRY(obj.add("ipv4_netmask", ipv4_netmask->view()));
             }
-            if (!adapter.ipv4_gateway().is_zero()) {
-                auto ipv4_gateway = TRY(adapter.ipv4_gateway().to_string());
-                TRY(obj.add("ipv4_gateway", ipv4_gateway->view()));
-            }
+            TRY(routing_table().with([&](auto& list) -> ErrorOr<void> {
```

Comment:
Maybe we should just do this in ifconfig as this is the only user of this field currently...

Reply:
Hmm. Or even, take it a step further and just remove it from ifconfig as well? An interface should be able to have multiple gateways depending on configured routes.

Reply:
I kept it because of my other PR (#13259) which relies on `ifconfig` setting default gateway. But we can just use `route` there.